<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormClassify</class>
 <widget class="QWidget" name="FormClassify">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="btnClassify">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
  border-image: url(&quot;:/resource/top-btn82-classify-add-n@3x.png&quot;);
}

QPushButton:hover
{
  border-image: url(&quot;:/resource/top-btn82-classify-add-h@3x.png&quot;);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFavorite">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(&quot;:/resource/top-btn22-classify-like-n@3x.png&quot;);
}

QLabel:hover
{
  border-image: url(&quot;:/resource/top-btn22-classify-like-h@3x.png&quot;);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelRating">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(&quot;:/resource/top-btn22-classify-rank-n@3x.png&quot;);
}

QLabel:hover
{
  border-image: url(&quot;:/resource/top-btn22-classify-rank-h@3x.png&quot;);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelArtist">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(:/resource/top-btn82-classify-add-n.png);
}

QLabel:hover
{
  border-image: url(:/resource/top-btn82-classify-add-h.png);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGenre">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(:/resource/top-btn82-classify-add-n.png);
}

QLabel:hover
{
  border-image: url(:/resource/top-btn82-classify-add-h.png);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelComposer">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(:/resource/top-btn82-classify-add-n.png);
}

QLabel:hover
{
  border-image: url(:/resource/top-btn82-classify-add-h.png);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTemp">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(:/resource/top-btn82-classify-add-n.png);
}

QLabel:hover
{
  border-image: url(:/resource/top-btn82-classify-add-h.png);
}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTemp_2">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>81</width>
       <height>26</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(:/resource/top-btn82-classify-add-n.png);
}

QLabel:hover
{
  border-image: url(:/resource/top-btn82-classify-add-h.png);
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
